### ISSUE TITLE
Update example apps to populate nimble-table with initial data

### ISSUE THREAD

**Issue by mollykreis** [ni/nimble#1195]
## 🧹 Tech Debt
With #1186, we had to remove the initial data from the `nimble-table` in the Angular example app because accessibility issues with the table were causing the Lighthouse check to fail.

When #1151 is fixed, we should add the initial data back to the table and ensure that the Lighthouse checks are passing.
